edx_sga/__init__.py

```diff
@@ -2,4 +2,4 @@
 Module for StaffGradedAssignmentXBlock.
 """
 
-__version__ = '0.8.3'
+__version__ = '0.8.4'
```

Comment:
Can you revert the change to this file? We have tooling that handles version numbers, and we're likely to roll this change up with copy/paste bugfix, #254

Reply:
Done